### DISCUSSION TITLE
Folders on Repository

### DISCUSSION THREAD

**Discussion by FilimonEnc** [community/community#72717] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Add the ability to create folders in which I could store repositories. I want to make a distinction so that training repositories, work repositories, and personal repositories are divided into folders

**Comment by softwaredevelop** [comment]
Use submodules.
[Git Tools - Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules)
[git-submodule Documentation](https://git-scm.com/docs/git-submodule)
